stingray/pulse/pulsar.py

```diff
@@ -340,44 +339,6 @@ def z_n(phase, n=2, norm=1):
                 for k in range(1, n + 1)])
 
 
-def fftfit_fun(profile, template, amplitude, phase):
```

Comment:
Do we need any backwards compatibility, or are the functions deleted here essentially not used by outside users?

Reply:
Not used!